scaffolding-ruby/lib/scaffolding.sh

```diff
@@ -393,7 +393,16 @@ scaffolding_create_dir_symlinks() {
 }
 
 scaffolding_create_files_symlinks() {
-  return 0
+  local entry source_file target
+
+  for entry in "${scaffolding_symlinked_files[@]}"; do
+    source_file="$scaffolding_app_prefix/$entry"
+    target="$pkg_svc_path/$entry"
```

Comment:
Now that I'm looking at this again after having recovered from chef-conf I just realized there are some issues with how this would work.

At the moment if `scaffolding_symlinked_files` has an entry for  `config/secrets.yml` it would link `app/config/secrets.yml` to `$pkg_svc_path/config/secets.yml` which works great because the supervisor automatically creates and manages creating those `config` files when starting the service.

In order to link those files into `var/`, I would need to know that I have to create my own `init` hook that copies those files to `$pkg_svc_var_path/config/secrets.yml`. Which seems a little odd to me especially since the supervisor could already create that file for me in `$pkg_svc_path/config`

`files/` could work since I can use `hab file upload` to add what ever I want there, but I would need to strip off the path since I don't see a way for that to create directories in `$pkg_svc_files_path`.  So this would create symlink that would look like `$scaffolding_app_prefix/config/secrets.yml` -> `$pkg_svc_files_path/secrets.yml`

Maybe a slightly more limited but straight forward option would be to have a `scaffolding_symlinked_configs` array.  It would take entries like `database.yml` and `secrets.yml` and would be strictly for linking the config file generated by the supervisor to a `config/` directory in the pkg `app/config` directory.  If you need to get more complicated than that for symlinking files around then that's when you would make your custom `init` hook.